pkg/workflow/v1/schema.go

```diff
@@ -0,0 +1,19 @@
+package v1
+
+import (
+	"github.com/puppetlabs/nebula-sdk/pkg/workflow/asset"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+var WorkflowSchema *gojsonschema.Schema
+
+func init() {
+	workflowSchemaLoader := gojsonschema.NewStringLoader(asset.MustAssetString("schemas/v1/Workflow.json"))
```

Comment:
You might be able to use `typeutil.LoadSchemaFromStrings` to make this a little cleaner. It validates the schema against the metaschema at boot too.